### DISCUSSION TITLE
Introductions!

### DISCUSSION THREAD

**Discussion by edakturk14** [BuidlGuidl/batch8.buidlguidl.com#1] [Announcements]
👋 Welcome everyone!

It's Eda here, I've been part of the BuidlGuidl for almost 2 years! I'm a developer and I spend a lot of time [writing](https://mirror.xyz/edatweets.eth) about different topics and concepts in web3. I spent most of my time working on developer education and creating content/documentation.

This is the 8th batch!! Hope that it will be a very fun project for you all to go through. If you have any questions, feel free to ask anytime!

**Comment by rohann06** [comment]
Hello everyone! I'm Rohan🙋🏻‍♂️.

I started as a front-end developer, and I've grown into someone who simply enjoys building regardless of the tech stack or language. I enjoy solving complex problems and am eager to dive deeper into the Web3 world.

 I'm really looking forward to learning, growing, and connecting with like-minded individuals in this batch. I’m excited about the potential collaborations and the opportunity to work with some truly talented devs and mentors.

**Reply by derrekcoleman** [reply]
Welcome, Rohan! Your spirit of learning and buidling is a great fit here in BuidlGuidl!

**Comment by RafaelCaso** [comment]
Hello! My name is Adrian and I've recently been exploring Web3 and learning solidity. I went through a bootcamp with Encode Club and then a Hackathon hosted by Dymension where I received a tier-3 prize for my entry. I've been using Scaffold-Eth a fair amount and it's definitely a very useful tool so I'm looking forward to learning more.

**Reply by derrekcoleman** [reply]
I LOVE Encode Club 😍 

Congrats on your hackathon prize! 💪 Keep it up and you'll win plenty more with SE-2

**Comment by ruhneb2004** [comment]
Hello peeps I'm Benhur😃
I am a second-year computer science student from India. While I have a solid understanding of Solidity, I am looking to improve my frontend development skills. I’m excited to better my development skills and collaborate with all of you. Once again, I’m glad to be here and looking forward to the super exciting stuffs! 🤩

**Reply by derrekcoleman** [reply]
Welcome, Benhur! The good news is most of the issues in this repo are about reading and displaying blockchain data using SE-2's hooks. In other words, a great chance to practice your frontend skills!

**Comment by CJValdez1** [comment]
Hi, I'm CJ Valdez. I'm an educator at a university in Mexico and I want to consolidate my knowledge in Solidity to train my students so that, in the future, they can also participate in challenges like SpeedrunEthereum and have the opportunity to become builders, thus continuing their academic development.
In the crypto world, I'm a SoloHomeStaker, as my background is in technology and I combine my work as an educator with my initial phase of building my Web3 curriculum. I’m very excited to be part of this community. I'm not sure how much time we have to solve this, but I hope to manage my time well, especially since classes start in a couple of weeks. I appreciate the opportunity and send my regards to everyone."

**Reply by derrekcoleman** [reply]
Hey CJ! From one educator to another, I appreciate the work you're doing 🫡 

Send your students to SpeedRunEthereum - I'd to see them all in our batches soon!

**Comment by indralukmana** [comment]
Hi everyone 👋

I'm Indra. I just finished the check-in issue. I've been a web2 front-end engineer for the past 6 year. My works mostly are in the React ecosystems but I also work in back-end and some docker/deployment stuff.

I have been interested in plunging into web3 devs and I found scaffoldeth a great way to bootstrap project prototypes and accelerate my learning. Excited to join everyone here in the web3 journey 🚀

**Reply by derrekcoleman** [reply]
Welcome, Indra! We always need more skilled (front-end) devs in web3. Excited to see what you buidl!

**Comment by 0xzeze** [comment]
Hello,

My name is 0xzeze . I love web3 because of the anonymous and freedom. I am computer science student in Europe. I am happy to be here!

**Reply by derrekcoleman** [reply]
Welcome, 0xzeze! Many of us are here because we share those same values.

**Comment by umutarray** [comment]
Hello i am Umut,
i am second year computer science student from Türkiye. I am interested in blockhain and it's technology and I have been in crypto since 2020

**Reply by derrekcoleman** [reply]
Hello Umut! Thank you for joining us.

**Comment by Selpak99** [comment]
Hello ,It is Selpak.  
I'm actually quite new in the blockchain. I watched playlist on Buidl Guidl's YouTube channel. They helped me a lot to learn fundamentals. I hope I can complete this batch. Thank you

**Reply by derrekcoleman** [reply]
Welcome, Selpak! So glad to hear the YouTube playlist was useful for you. We look forward to your contributions!

**Comment by penaMaster** [comment]
Good afternoon everyone!

I’m Ali, and I’m happy to be here today. I like playing the guitar, it’s a fun hobby for me.

I’m also interested in cryptocurrency. I like trading crypto and reading whitepapers about new projects. Learning more about the crypto world is something I enjoy a lot.

Thank you

**Reply by derrekcoleman** [reply]
Hi Ali 🎸 

You're in the right place! After this Batch you'll know a bit more about how to contribute to any crypto project you're interested in!

**Comment by MexicanAce** [comment]
Hi everyone 👋 !

I'm Nick and I'm currently a Software Engineer for Matter Labs (creators of ZKsync) and lead the DevEx/DevRel/UX team. A few projects I've worked include In-Memory Node for ZKsync, SDKs, Portal/Bridge, custom Block Explorer, and ZK Quest (a learn to earn website where devs can complete coding challenges to earn real ETH at in-person conferences).

I've always heard about Buidl Guidl and I finally dedicated some time to complete the speedrun and join 🙌 ! Glad to be here on the journey with y'all!!
